app/helpers/header-date.js

```diff
@@ -3,7 +3,7 @@ import moment from 'moment';
 
 export function headerDate(params) {
   const timezone = params[1] ? params[1] : moment.tz.guess();
-  return `${moment(params[0]).tz(timezone).format('dddd, D MMMM, YYYY h:mm A')} (${moment.tz(params[0], timezone).zoneAbbr()})`;
+  return `${moment(params[0]).tz(timezone).format('dddd, MMMM Do YYYY, h:mm A')} (${moment.tz(params[0], timezone).zoneAbbr()})`;
```

Comment:
Why reverted the format here? We need same format everywhere

Reply:
This was the only way for the build to be successful.
Changing it was failing the build every time.

Reply:
No, that's not the only way to fix the build